maintainers/scripts/activity/activity.bb

```diff
@@ -0,0 +1,425 @@
+#!/usr/bin/env bb
+
+;; to get a repl
+;; - Terminal
+;;       $ nix-shell -p babashka curl --run "bb"
+;;       user=> (load-file "activity.bb")
+;;       user=> (pprint (file-query :org-repos {:org "nixos"}))
+;; - IDE support
+;;       $ nix-shell -p babashka curl --run "bb nrepl-server"
+;;   - connect to 127.0.0.1:1667 with emacs/cider or vscode/calva
+
+;; # Configuration
+
+(require '[babashka.curl :as curl]
+         '[babashka.cli :as cli])
+
+(def wd (.getParentFile (fs/file *file*)))
+
+(def graphql-path
+  "Make working directory overridable"
+  (str
+   (or (System/getenv "NIXOS_ACTIVITY_GQL_PATH")
+       wd)))
+
+(def maintainers-path
+  "Make maintainers path overridable"
+  (str
+   (or (System/getenv "NIXOS_MAINTAINERS_PATH")
+       (fs/canonicalize
+        (fs/file wd "../..")))))
+
+;; (println "graphql path:" graphql-path)
+
+(defn get-access-token
+  "GitHub access token lookup"
+  []
+  (or (System/getenv "ACCESS_TOKEN")
+      (try
+        (slurp (fs/file (fs/home) ".config/nix/maintainer-activity-access-token"))
```

Comment:
I don't really like that this script is reusing the `~/.config/.nix` from `nix` itself for another tool. Can we use another location?

Reply:
Do you have suggestions for other locations? `~/.config/nixpkgs` maybe?

Reply:
moved to `~/.config/nixpkgs/maintainer-activity-access-token`